xrdp/xrdp_login_wnd.c

```diff
@@ -401,7 +401,7 @@ xrdp_wm_show_edits(struct xrdp_wm *self, struct xrdp_bitmap *combo)
                 set_string(&b->caption1, name);
 
                 /* edit */
-                b = xrdp_bitmap_create(DEFAULT_EDIT_W, DEFAULT_EDIT_H, self->screen->bpp,
```

Comment:
please rm `DEFAULT_EDIT_W` if is unused

Reply:
DEFAULT_COMBO_W was also unused. I force-pushed the amended commit.

Reply:
great